flink-end-to-end-tests/test-scripts/test_shaded_hadoop_s3a.sh

```diff
@@ -22,15 +22,11 @@
 source "$(dirname "$0")"/common.sh
 source "$(dirname "$0")"/common_s3.sh
 
-s3_put $TEST_INFRA_DIR/test-data/words $ARTIFACTS_AWS_BUCKET flink-end-to-end-test-shaded-s3a
-# make sure we delete the file at the end
-function shaded_s3a_cleanup {
-  s3_delete $ARTIFACTS_AWS_BUCKET flink-end-to-end-test-shaded-s3a
-}
-trap shaded_s3a_cleanup EXIT
-
 start_cluster
 
+bucket=flink-e2e-tests
```

Comment:
Could we move the definition of `bucket` to `common_s3.sh`? Also I think we should use uppercase here to follow the convention of constants like: `TEST_INFRA_DIR`, `ARTIFACTS_AWS_BUCKET` etc.